Formula/docker-machine-driver-xhyve.rb

```diff
@@ -23,6 +23,8 @@ class DockerMachineDriverXhyve < Formula
     depends_on "libev"
   end
 
+  patch :DATA
```

Comment:
Can you please add a link to the upstream patch commit and explanation of why the patch is needed in a comment in the formula file. Thanks!